index.d.ts

```diff
@@ -0,0 +1,65 @@
+declare module 'addsearch-js-client' {
+    export default class AddSearchClient {
+      constructor(sitekey: string, privatekey?: string);
+  
+      search(keyword: string, callback: (results: any) => void): void;
+      search(callback: (results: any) => void): void;
```

Comment:
I'm not sure about this: does it mean typescript can distinguish scenarios when `search` is called with 1 argument and when it is called with 2 arguments?
Or does the second definition of `search` overrides the previous one?